### ISSUE TITLE
Error executing  export-to-gml.scm example

### ISSUE THREAD

**Issue by elymatos** [opencog/generate#12]
Hi,
I'm getting an error when executing the export-to-gml example at this step:

```
;; Convert the Atomese representation of these networks into
;; a GML string.
(define gml-string (export-to-gml graph-set))
```

```
scheme@(guile-user) [1]> (define gml-string (export-to-gml graph-set))
ice-9/boot-9.scm:1685:16: In procedure raise-exception:
Value out of range 0 to 18446744073709551615: -1

```
Some direction about to to fix that?
Thanks,
Ely

**Comment by elymatos** [comment]
The error appears to be at `cog-assign-uuid`

```
scheme@(guile-user) [1]> (cog-assign-uuid (Concept "e"))
ice-9/boot-9.scm:1685:16: In procedure raise-exception:
Value out of range 0 to 18446744073709551615: -1

```

**Comment by linas** [comment]
Yep. Appears that guile no longer allows -1 to be cast to uintmax I just fixed this in opencog/atomspace commit 52aa0079a5e580cc3592d9cf087b8dfdb8d189fe

You'll have to pull a new atomspace and rebuild it to get the fix.

Sorry about the bugs, this git repo has not been high-priority recently.

**Comment by linas** [comment]
The -1 value meant "I don't have a UUID yet, give me a new one"
